deploy/add_external_env.sh

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
This file is they part. It's a bash script to import your current AWS cloudformation stack as a new environment. Environments are a lot more flexible and stored correctly in git